src/main/java/stringsum/StringSum.java

```diff
@@ -0,0 +1,66 @@
+package stringsum;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringSum {
+
+    public int basicSum(String testData){
+        if(testData == null || testData.isBlank() || testData.trim().isEmpty()){
```

Comment:
[자바 공식문서](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html)에서 String.isBlank 메서드는 아래와 같이 설명되어있어요.
Returns true if the string is empty or contains only [white space]codepoints, otherwise false.

testData.isBlank() 메서드와 testData.trim().isEmpty() 메서드가 중복이지 않을까해요!